README.md

```diff
@@ -1,7 +1,9 @@
 # Community Sops Collection
+
```

Comment:
Please do not modify unrelated parts of the README.

Reply:
Sorry, that was my auto formatter